dev/com.ibm.ws.security.authentication.tai/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -59,3 +59,6 @@ extraProperties.description=Collection of properties for the interceptor.
 disableLtpaCookie=Disable LTPA cookie
 disableLtpaCookie.desc=Do not create an LTPA cookie for the trust association interceptor (TAI). 
 
+initAtFirstRequest=Initialize the TAI at the first incoming request
+initAtFirstRequest.desc=Control whether to initialize the trust association interceptor (TAI) at the first incoming request or at the server start up. The default is initializing the TAI at the server start up. 
```

Comment:
Change both uses of "start up" to "startup" (one word). The term is used a noun both times in this message description so is "startup."

Also, consider replacing "Control" with "Choose" or "Select" and "is initializing" with "to initialize."

Reply:
Something likes below?
Select whether to initialize the trust association interceptor (TAI) at the first incoming request or at the server startup. The default is to initialize the TAI at the server startup.